src/e2e-test/features/gcs/source/GCSSourceToBigQuery.feature

```diff
@@ -285,3 +285,304 @@ Feature: GCS source - Verification of GCS to BQ successful data transfer
+    Then Close the preview
+    Then Deploy the pipeline
+    Then Run the Pipeline in Runtime
+    Then Enter runtime argument value "projectId" for key "gcsProjectId"
+    Then Enter runtime argument value "serviceAccountType" for key "serviceAccountType"
+    Then Enter runtime argument value "serviceAccount" for key "serviceAccount"
+    Then Enter runtime argument value "gcsCsvFile" for GCS source property path key "gcsSourcePath"
+    Then Enter runtime argument value "gcsSkipHeaderTrue" for key "gcsSkipHeader"
+    Then Enter runtime argument value "csvFormat" for key "gcsFormat"
+    Then Enter runtime argument value "sampleSize" for key "SampleSize"
+    Then Enter runtime argument value "gcsOverrideField" for key "OverRide"
+    Then Enter runtime argument value "gcsMinSplitSize" for key "MinSplit"
+    Then Enter runtime argument value "gcsMaxSplitSize" for key "MaxSplit"
+    Then Enter runtime argument value "fileRegex" for key "FileReg"
+    Then Enter runtime argument value "filenameOnly" for GCS source property path key "FilenameOnly"
+    Then Enter runtime argument value "recursive" for GCS source property path key "ReadFilesRecursively"
+    Then Enter runtime argument value "ignoreNonExistingFolders" for GCS source property path key "IgnoreNonExistingFolders"
+    Then Enter runtime argument value "encrypted" for GCS source property path key "DataFileEncrypted"
+    Then Enter runtime argument value "encryptedMetadataSuffix" for GCS source property path key "testmeta"
+    Then Enter runtime argument value "gcsFileSysProperty" for key "FileSystemPr"
+    Then Enter runtime argument value "fileEncoding" for key "Encode"
+    Then Enter runtime argument value "gcsCSVFileOutputSchema" for key "gcsOutputSchema"
+    Then Enter runtime argument value "projectId" for key "bqProjectId"
+    Then Enter runtime argument value "projectId" for key "bqDatasetProjectId"
+    Then Enter runtime argument value "dataset" for key "bqDataset"
+    Then Enter runtime argument value for BigQuery sink table name key "bqTargetTable"
+    Then Enter runtime argument value "cmekBQ" for BigQuery cmek property key "cmekBQ" if BQ cmek is enabled
+    Then Enter runtime argument value "bqTruncateTableTrue" for key "bqTruncateTable"
+    Then Enter runtime argument value "bqUpdateTableSchemaTrue" for key "bqUpdateTableSchema"
+    Then Run the Pipeline in Runtime with runtime arguments
+    Then Wait till pipeline is in running state
+    Then Open and capture logs
+    Then Verify the pipeline status is "Succeeded"
+    Then Get count of no of records transferred to target BigQuery Table
+    Then Validate the cmek key "cmekBQ" of target BigQuery table if cmek is enabled
+
+  @GCS_OUTPUT_FIELD_TEST @BQ_SINK_TEST
+  Scenario: To verify successful data transfer from GCS to BigQuery with macro Path Field
+    Given Open Datafusion Project to configure pipeline
+    When Select plugin: "GCS" from the plugins list as: "Source"
+    When Sink is BigQuery
```

Comment:
Use the new step similar to above for selecting the Bigquery plugin from sink

Reply:
Done.